tests/acceptance/features/webUIFiles/copyPrivateLinks.feature

```diff
@@ -9,11 +8,7 @@ Feature: copy current path as a permanent link
     And user "user1" has logged in using the webUI
     And the user has browsed to the files page
 
-  Scenario Outline: Copy permalink to clipboard
```

Comment:
Why removing the scenario outline ?

Reply:
Because previously there was the folder named "/" which is not a valid folder name. So we have only one `folder_name` to check. Thus I thought there is no need for scenario outline.

Reply:
that feature was changed in https://github.com/owncloud/phoenix/pull/2496